stock_barcodes/wizard/stock_barcodes_read.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2019 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import _, api, fields, models
+from odoo.addons import decimal_precision as dp
+
+
+class WizStockBarcodesRead(models.TransientModel):
+    _name = 'wiz.stock.barcodes.read'
+    _inherit = 'barcodes.barcode_events_mixin'
+    _description = 'Wizard to read barcode'
+    # To prevent remove the record wizard until 2 days old
+    _transient_max_hours = 48
```

Comment:
I don't see it very predictive. If inventory is done on more than 2 days... 
I have no magic solution, except move the model into models.Model, and purge the database regularly, but only when the res_model_id is in a done state. What do you think ?
After all, a data keeped during 48 hours is not very transient, doesn't it ?

Reply:
The inventory process can last 2, 3 or n days, but the wizard opened from inventory is new each time that user open the scan wizard interface. I put this time to avoid that in the same day with the  same wizard opened the autovacumm deletes this wizard, I am sure that the next day the user continues to do the inventory but he crate a new wizard...
As important data, the scan logs records are linked to inventory (res_model, res_id) in this case.